actionners/actionners.go

```diff
@@ -129,6 +131,15 @@ func GetDefaultActionners() *Actionners {
 				CheckParameters: nil,
 				Action:          k8sDelete.Action,
 			},
+			&Actionner{
+				Category:        "aws",
+				Name:            "lambdainvoke",
```

Comment:
```suggestion
				Name:            "lambda",
```

I prefer this name, wdyt?

Reply:
If we'd have only a single action regarding lambda I agree, but I was thinking if we plan to add other actions in the future, for example deletelambda. Would be something similar to the k8s actions, wdyt?

Reply:
Given a falco rule triggered by a lambda that wasn't supposed to be invoked in a certain way we might want to delete it, does it make sense?

Reply:
I was thinking about an `aws:delete` like I did for `kubernetes:delete`

Reply:
That makes sense, the function would need to be insanely generic though, kind of the same approach you used [here](https://github.com/Falco-Talon/falco-talon/blob/a623905e101a706f8639d0e93ccedb3a5be22c39/actionners/kubernetes/delete/delete.go#L33) but for more resources.